vault/resource_pki_secret_backend_config_issuers.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+var pkiSecretBackendFromConfigIssuersPathRegex = regexp.MustCompile("^(.+)/config/issuers")
+
+const (
+	fieldDefaultFollowsLatestIssuer = "default_follows_latest_issuer"
+)
+
+func pkiSecretBackendConfigIssuers() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: provider.MountCreateContextWrapper(pkiSecretBackendConfigIssuersCreateUpdate, provider.VaultVersion111),
+		UpdateContext: pkiSecretBackendConfigIssuersCreateUpdate,
+		DeleteContext: pkiSecretBackendConfigIssuersDelete,
+		ReadContext:   provider.ReadContextWrapper(pkiSecretBackendConfigIssuersRead),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			consts.FieldBackend: {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
```

Comment:
I'm not familiar with the PKI secret engine so maybe this question doesn't actually make sense, but is there a reason this path uses ForceNew and can't be migrated if a user was to first migrate the PKI secret engine backend itself?

Reply:
Ah that's a nice catch! I copied over some of the patterns from other PKI resources that currently can not be migrated by the TFVP and forgot to update that here. I tested this resource with mount migration and looks like it does not need the `ForceNew` on the backend and it can be migrated. Thanks!